cli/src/pcluster/config/cluster_config.py

```diff
@@ -1143,6 +1144,8 @@ def __init__(
         self.config_version = ""
         self.original_config_version = ""
         self._official_ami = None
+        # # Cache subnet ids information together to reduce number of boto3 calls
+        # AWSApi.instance().ec2.describe_subnets(subnet_ids=self.compute_subnet_ids)
```

Comment:
I wanted to keep this visible, since it didn't work and I didn't have time to dive deeper right now.  Basically, the objects we need don't exist yet, so not sure if this is a viable path.